index.js

```diff
@@ -86,15 +90,18 @@ import typeof useWindowDimensions from './Libraries/Utilities/useWindowDimension
 import typeof UTFSequence from './Libraries/UTFSequence';
 import typeof Vibration from './Libraries/Vibration/Vibration';
 import typeof YellowBox from './Libraries/YellowBox/YellowBoxDeprecated';
-import typeof LogBox from './Libraries/LogBox/LogBox';
-import typeof RCTDeviceEventEmitter from './Libraries/EventEmitter/RCTDeviceEventEmitter';
-import typeof RCTNativeAppEventEmitter from './Libraries/EventEmitter/RCTNativeAppEventEmitter';
+
+// Plugins
```

Comment:
What's a plugin? I'd move these into `Modules`.

Reply:
Similar to above, there is a section in the module.exports for plugins, does this matter?

https://github.com/facebook/react-native/blob/9c881b0f9712baef7b6dd6eb530bf371b794fade/index.js#L448-L458